src/frontend/src/components/pinInput.ts

```diff
@@ -258,3 +284,19 @@ const focusNextOrBlur = (element: HTMLInputElement) => {
     element.blur();
   }
 };
+
+// Focus the previous input in the list, if any. Returns true if the focus was moved
+// to a previous input.
+const focusPrevious = (element: HTMLInputElement): boolean => {
```

Comment:
it does not just "focus" it also empty the value if I get it so it would be worth reflecting this in the method name for readability

Reply:
You're completely right; I added that at the last minute and forgot to update